hack/envtest/_matrix/v1.28.3.yaml

```
@@ -0,0 +1,2 @@
+go: 1.21
```

Should be 1.20: https://github.com/kubernetes/kubernetes/blob/v1.28.3/.go-version
```suggestion
go: 1.20
```

---

Good point. I would keep 1.21 here, as we already used it for 1.28.0

It wasn't correct, but it would be strange to "downgrade" in a higher version now